content/es/docs/concepts/services-networking/network-policies.md

```diff
@@ -135,7 +135,7 @@ combinaciones de plugin de red, proveedor de nube, implementación de `Service`,
 
 En el caso de la entrada, esto significa que en algunos casos se pueden filtrar paquetes 
 entrantes basándose en la IP de origen real, mientras que en otros casos, la "IP de origen" sobre la que actúa la
-la NetworkPolicy actúa puede ser la IP de un `LoadBalancer` o la IP del Nodo donde este el Pod involucrado, etc.
+NetworkPolicy puede ser la IP de un `LoadBalancer` o la IP del Nodo donde está el Pod involucrado, etc.
```

Comment:
If you can, avoid starting a sentence with an API kind (eg “NetworkPolicy”). It's OK to reword so long as the overall meaning matches the English original.